platform/security/sts/security-sts-ldaplogin/src/main/java/ddf/ldap/ldaplogin/SslLdapLoginModule.java

```diff
@@ -270,6 +271,10 @@ protected boolean doLogin() throws LoginException {
                         LOGGER.info("User {} not found in LDAP.", user);
                         return false;
                     }
+                    if (entryReader.isReference()) {
```

Comment:
❓ I wonder if we shouldn't iterate in this case. There could be a user that matches the filter in the local ldap but they're after this reference in the results we got back. Something like...
```java
while (entryReader.hasNext() && entryReader.isReference()) {
    entryReader.readReference();
}
if (!entryReader.hasNext()) {
    LOGGER.info("User {} not found in LDAP.", user);
    return false;
}
SearchResultEntry searchResultEntry = entryReader.readEntry();
```